core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/controller/Controller.scala

```diff
@@ -332,18 +332,40 @@ class Controller(
       .sum
   }
 
+  // the output count is the sum of the output counts of the last-layer actors
+  private def aggregateWorkerOutputResults(
```

Comment:
keep in mind this part of the code is no longer valid in the current codebase.

Reply:
updated to follow the new codebase